### PR TITLE
GitHubSync update - release-6.0

### PR DESCRIPTION
**Pull request by particularbot6**
This is an automated synchronization PR.

The following source template repositories were used:
* particular/RepoStandards

### CODE REVIEW

File: src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs [hunk 1]
```diff
@@ -204,8 +204,9 @@ public class SagaData : ContainSagaData
 
         protected override string CorrelationPropertyName
         {
-            //Explicitly not use expression body
+#pragma warning disable IDE0025 // Use expression body for properties
```
Comment (SeanFeldman) [inline src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs:207]: Should we keep the comment as it was intentionally done this way?

Comment (bording) [reply src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs:207]: It's redundant with the new comment, so I removed it.

Comment (SeanFeldman) [reply src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs:207]: I'm not sure. There's a difference between reading
> // Explicitly not use expression body

vs

> #pragma warning disable IDE0025 // Use expression body for properties

Comment (bording) [reply src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs:207]: We're having the same conversation of two different PRs.

The name of the class in the test already conveys that it's expecting a property with a body.

Comment (bording) [reply src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs:207]: I can add it back if you really think it's important.

**Comment by bording** [comment]
Not sure why the one test leg failed with an approval file problem when the others didn't. The test has already been properly updated to exclude the attribute that seems to have shown up to fail the test.

I'll have to investigate more tomorrow.

File: src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs (hunk 1 above)
Comment (SeanFeldman) [reply src/ScriptBuilder.Tests/Saga/SagaDefinitionReaderTest.cs:207]: I think that'd be helpful. Thanks 🙂

**Review by mikeminutillo — APPROVED** [review]

**Review by SeanFeldman — APPROVED** [review]

**Pull request merged**